internal/cli/commands.go

```diff
@@ -613,11 +616,55 @@ func (c *HelpCommand) Execute(ctx context.Context, ee *ExecutionEnvironment) (*E
 	return result, nil
 }
 
+// ----------------------------------------------------------------------------
+// Submit Command
+// ----------------------------------------------------------------------------
+
+// ConnectCommand is a command that connects to an RPC endpoint
+type SubmitCommand struct {
+	Transaction string
+}
+
+// NewConnectCommand creates a new connect object
+func NewSubmitCommand(inv *CommandParseResult) Command {
+	return &SubmitCommand{Transaction: *inv.Args["transaction"]}
+}
+
+// Execute connects to an RPC endpoint
+func (c *SubmitCommand) Execute(ctx context.Context, ee *ExecutionEnvironment) (*ExecutionResult, error) {
+	result := NewExecutionResult()
+
+	if !ee.IsOnline() {
+		return nil, fmt.Errorf("%w: cannot submit transaction", cliutil.ErrOffline)
+	}
+
+	// Unmarshal the transaction
+	data, err := base64.StdEncoding.DecodeString(c.Transaction)
```

Comment:
Do we want to use std encoding or url encoding?

Our base64 encodings inside of protobuf messages are url base64